cluster/gce/manifests/etcd.manifest

```diff
@@ -65,7 +72,7 @@
         "command": [
           "/bin/sh",
           "-c",
-          "set -x; exec /usr/local/bin/etcdctl --endpoints=127.0.0.1:{{ port }} {{ etcdctl_certs }} --command-timeout=15s endpoint health"
+          "set -x; exec /usr/local/bin/etcdctl-{{ pillar.get('etcd_version', '3.4.13') }} --endpoints=127.0.0.1:{{ port }} {{ etcdctl_certs }} --command-timeout=15s endpoint health"
```

Comment:
ditto. E.g. listed binaries in the 3.4.9 (gcr.io/google-containers/etcd-amd64:3.4.9) image:

```
-r-xr-xr-x  1 ptab primarygroup 23827424 Jun 25  2020 etcd
-r-xr-xr-x  1 ptab primarygroup 20186048 Jun 25  2020 etcd-3.0.17
-r-xr-xr-x  1 ptab primarygroup 16352288 Jun 25  2020 etcd-3.1.12
-r-xr-xr-x  1 ptab primarygroup 17899872 Jun 25  2020 etcd-3.2.24
-r-xr-xr-x  1 ptab primarygroup 22102784 Jun 25  2020 etcd-3.3.17
-r-xr-xr-x  1 ptab primarygroup 23827424 Jun 25  2020 etcd-3.4.9
-r-xr-xr-x  1 ptab primarygroup 17612384 Jun 25  2020 etcdctl
-r-xr-xr-x  1 ptab primarygroup 18468640 Jun 25  2020 etcdctl-3.0.17
-r-xr-xr-x  1 ptab primarygroup 14319264 Jun 25  2020 etcdctl-3.1.12
-r-xr-xr-x  1 ptab primarygroup 15279616 Jun 25  2020 etcdctl-3.2.24
-r-xr-xr-x  1 ptab primarygroup 17770784 Jun 25  2020 etcdctl-3.3.17
-r-xr-xr-x  1 ptab primarygroup 17612384 Jun 25  2020 etcdctl-3.4.9
```

Reply:
Fixed.